snuba/settings/__init__.py

```diff
@@ -273,7 +274,7 @@ class RedisClusters(TypedDict):
 
 # The migration groups that can be skipped are listed in OPTIONAL_GROUPS.
 # Migrations for skipped groups will not be run.
-SKIPPED_MIGRATION_GROUPS: Set[str] = set()
+SKIPPED_MIGRATION_GROUPS: Set[str] = {"group_attributes"}
```

Comment:
Adding the migration to this set will not be necessary. The readiness state will handle whether or not the migration is executed in specific environments.